src/main/java/io/apicurio/datamodels/cmd/commands/DeleteExampleCommand_30.java

```diff
@@ -35,7 +35,7 @@
  * @author eric.wittmann@gmail.com
  */
 public class DeleteExampleCommand_30 extends AbstractCommand {
-
+    // TODO: Ordering on undo
```

Comment:
Can you make this TODO more descriptive? It's just so that if you don't have time to do this, someone else can look at it and understand what you mean. Perhaps a GitHub issue would be better.

Reply:
Yeah these were just for me to come back to. These are the commands that I think still need some attention but I haven't gone back to look at them since I was waiting for a bit of feedback on the general approach. I'll probably get back to it next week.